std/installer/test.ts

```diff
@@ -12,6 +12,9 @@ import { install, isRemoteUrl } from "./mod.ts";
 let fileServer: Deno.Process;
 const isWindows = Deno.build.os === "win";
```

Comment:
I didn't get a chance to change this because the code wasn't there, but could you use `path.isWindows` instead?

Reply:
if i get it working :)